packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -1,145 +1,342 @@
-// Copyright 2015 The Chromium Authors. All rights reserved.
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
I didn't think we did this?

Reply:
No, we don't.  That's a remnant of me implementing a new API, and then shoehorning it into the old one.  Reverted to the 2015 date.